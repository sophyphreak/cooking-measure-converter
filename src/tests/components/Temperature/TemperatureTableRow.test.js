import React from 'react';
import { shallow } from 'enzyme';
import TemperatureTableRow from '../../../components/Temperature/TempertureTableRow';

let metricLabel, metricInput, imperialInput, imperialLabel, wrapper;

beforeEach(() => {
  metricLabel = 'C';
  metricInput = { onChange: jest.fn(), value: '' };
  imperialInput = { onChange: jest.fn(), value: '' };
  imperialLabel = 'F';
  wrapper = shallow(
    <TemperatureTableRow
      metricLabel={metricLabel}
      metricInput={metricInput}
      imperialInput={imperialInput}
      imperialLabel={imperialLabel}
    />
  );
});

test('should render TemperatureTableRow correctly', () => {
  expect(wrapper).toMatchSnapshot();
});
