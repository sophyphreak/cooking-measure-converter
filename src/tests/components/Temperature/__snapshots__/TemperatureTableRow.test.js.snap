// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render TemperatureTableRow correctly 1`] = `
<tr>
  <th>
    <Label
      className="align-right"
      tag="label"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    >
      C
    </Label>
  </th>
  <td>
    <AmountInput
      onChange={[MockFunction]}
      value=""
    />
  </td>
  <td>
    <AmountInput
      onChange={[MockFunction]}
      value=""
    />
  </td>
  <th>
    <Label
      tag="label"
      widths={
        Array [
          "xs",
          "sm",
          "md",
          "lg",
          "xl",
        ]
      }
    >
      F
    </Label>
  </th>
</tr>
`;
